BUILD.md

```diff
@@ -24,22 +24,28 @@ cmake -H. -Bbuild
 make -C build -j3
 ```
 
-Then you can install it into your virtualenv:
+This build directory does not have to be identical to the `build` directory
+created by `./configure` as it is here.
+
+CMake `POST_BUILD` hooks on shared libraries targets will handle installing the
+dev tree into your virtualenv.
 
-```shell
-pip install -e .
-```
 ### Running Unit Tests
 
-You can run all the unite tests using the following command.
+You can run all the unit tests not marked as slow using the following command.
 
 ```shell
-pytest -rfs -m "not slow" coremltools/test
+make -j3 -C build pytest_no_slow
```

Comment:
shouldn't I still be able to run the non slow tests with the usual `pytest` command?

Reply:
Good point. It would be good to say what the command runs because sometimes people want to run a specific unit test to not the whole setup. So it would be good to be explicit about what the workflow would be for folks that want to run a single test.

Reply:
Of course you can, but this depends on recompiling automatically.

Reply:
So does that I mean I need to make the build (make -j3 -C build) and then run the test using pytest per usual.

Reply:
I would update those instructions accordingly.

Reply:
I've clarified this section of the BUILD.md